includes/helper/class-wp-job-manager-helper-listing-renewal.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * File containing the class WP_Job_Manager_Helper_Listing_Renewal.
+ *
+ * @package wp-job-manager
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Handles the renewal of Job Listings.
+ *
+ * @since $next-version$$
```

Comment:
We don't currently replace these during our releases. Maybe we should just use the next version for now?

Reply:
Yeah been thinking about that. Going with 1.41.0 for now since it's a new feature release. I had `$$next-version$$` in other places too. https://github.com/Automattic/WP-Job-Manager/pull/2422/commits/df5a472d8a6edf285e5f2155b36af6235b2727d5